scripts/build

```
@@ -493,19 +490,43 @@ if [ "${TARGET}" = "target" -o "${TARGET}" = "init" ]; then
 
     python_fix_abi "${INSTALL}"
 
-    if [ ! "${BUILD_WITH_DEBUG}" = "yes" ]; then
-      ${STRIP} $(find ${INSTALL} \
-        -type f -name "*.so*" \
-        ! -name "ld-*.so" \
-        ! -name "libc-*.so" \
-        ! -name "libpthread-*.so" \
-        ! -name "libthread_db-*so" \
-        2>/dev/null) 2>/dev/null || :
-        if [ "${TARGET}" = "init" ]; then
-          ${STRIP} $(find ${INSTALL} -type f -name "*.so*" 2>/dev/null) 2>/dev/null || :
-        fi
-        ${STRIP} $(find ${INSTALL} ! -name "*.so*" ! -name "*.ko" \
-          -type f -executable 2>/dev/null) 2>/dev/null || :
+    if [ -d "${INSTALL}" ]; then
+
+      if [ "${TARGET}" = "target" ]; then
+
+        FILES_TO_STRIP="$(find ${INSTALL} \
+          -type f -executable \
+          ! -name "*.ko" \
+          ! -name "ld.so" \
+          ! -name "ld-*.so" \
+          ! -name "libc-*.so" \
+          ! -name "libpthread-*.so" \
+          ! -name "libthread_db-*so" \
```

Suggestion to strip all libs even without executable flag:
```suggestion
          -type f \( -executable -o -name "*.so*" \) \
          ! -name "*.ko" \
```